test/testInteraction.cpp

```diff
@@ -40,6 +40,8 @@ TEST(ElectronPairProduction, allBackgrounds) {
 	epp.setPhotonField(irb);
 	irb = new IRB_Stecker16_lower();
 	epp.setPhotonField(irb);
+    irb = new IRB_Finke22();
```

Comment:
Whitespace

Reply:
I am not creating a new commit just to fix the whitepace, but we should probably do this at some point in a consistent way for all files.